app/views/sites/edit.html.erb

```diff
@@ -15,24 +15,51 @@
 
 <h1 class="govuk-heading-xl"><%= @site.location_name %></h1>
 
+<% if @errors.any? then %>
```

Comment:
When there is an error can we also prefix the title with `Error: ` as recommended by https://design-system.service.gov.uk/components/error-summary/

Reply:
Done, good suggestion 👍